/*
 * hovergroup_proto
 *        File: acommsSim.proto
 *  Created on: Jan 13, 2014
 *      Author: Josh Leighton
 */

message AcommsSimReport {
    required string vehicle_name = 1;
    
    required double x = 2;
    required double y = 3;
    
    optional double heading = 4;
    optional double speed = 5;
    optional double depth = 6;
    
    required bool ranging_enabled = 7;
    
}
